types/adobe__pdfservices-node-sdk/operation/split-pdf-operation.d.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * An operation that splits PDF document into multiple smaller documents by simply specifying either the number of files,
+ * pages per file, or page ranges.
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        SplitPDF = PDFServicesSdk.SplitPDF,
+ *        splitPDFOperation = SplitPDF.Operation.createNew();
+ *
+ *  splitPDFOperation.setInput(PDFServicesSdk.FileRef.createFromLocalFile('~/Documents/splitPDFInput.pdf',
+ *  	SplitPDF.SupportedSourceFormat.pdf));
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/split-pdf-operation.d.ts;
  Line: 16;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.